client/my-sites/checkout/cart/cart-free-user-plan-upsell.jsx

```diff
@@ -56,48 +58,90 @@ class CartFreeUserPlanUpsell extends Component {
 	};
 
 	getUpgradeText() {
-		const { cart, planPrice, translate } = this.props;
+		const { cart, planPrice, translate, eligibleForProPlan } = this.props;
 		const firstDomain = find( getAllCartItems( cart ), this.isRegistrationOrTransfer );
+		const planName = eligibleForProPlan ? 'Pro' : 'Personal';
```

Comment:
Is it ok that these plan names are not translated? I don't know if we localize plan names or not but it seems like we might since "Personal" is translated in the previous versions of the strings below.